app.rb

```diff
@@ -0,0 +1,94 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  attr_accessor :books, :students, :teachers, :rentals
+
+  def initialize
+    @books = []
+    @students = []
+    @teachers = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each_with_index { |book, i| print "(#{i}) Title: \"#{book.title}\", Author: #{book.author}\n" }
+  end
+
+  def list_rentals
+    puts 'Enter person ID: '
+    user_input = gets.chomp.to_i
+    user_rentals = []
+    @rentals.each { |r| user_input == r.person.id ? user_rentals << r : '' }
+    print "Rentals: \n"
+    user_rentals.each { |r| print " Date: #{r.date}, Title: #{r.book.title}, Author: #{r.book.author}\n" }
+  end
+
+  def list_people
+    @students.each { |s| print "[#{s.class}] Name: #{s.name}, ID: #{s.id}, Age: #{s.age}\n" }
+    @teachers.each { |s| print "[#{s.class}] Name: #{s.name}, ID: #{s.id}, Age: #{s.age}\n" }
+  end
+
+  def create_person
+    print "Do you want to create: \n 1 - Student \n 2 - Teacher \n 3 - Cancel \n "
+    user_input = gets.chomp.to_i
+
+    case user_input
+    when 1
+      print ' Enter student age: '
+      age = gets.chomp.to_i
+      print ' Enter classroom: '
+      classroom = gets.chomp
+      print ' Enter name: '
+      name = gets.chomp
+      new_student = Student.new(age, classroom)
+      new_student.name = name unless name.empty?
+      @students << new_student
+      puts " #{new_student.name} added successfully"
```

Comment:
- [ ] Please ask for parent permission when creating a student as shown in the example video. It is important that our app behaves exactly like the example video 👍 
- [ ] An error occurs while either trying to create a new student or a new teacher. Please try to fix it 🐛 
![image](https://user-images.githubusercontent.com/46839548/189243663-c5c76e3c-1952-4b51-bf2f-6abd0d82213e.png)